packages/zipkin/src/instrumentation/httpClient.js

```diff
@@ -0,0 +1,46 @@
+const Annotation = require('../annotation');
+const Request = require('../request');
+
+function requiredArg(name) {
+  throw new Error(`HttpClientInstrumentation: Missing required argument ${name}.`);
+}
+
+class HttpClientInstrumentation {
+  constructor({
+    tracer = requiredArg('tracer')
+  }) {
+    this.tracer = tracer;
+  }
+
+  recordRequest(serviceName, request, remoteServiceName, url, method) {
+    this.tracer.setId(this.tracer.createChildId());
+    const traceId = this.tracer.id;
+
+    this.tracer.recordServiceName(serviceName);
```

Comment:
serviceName should be constant for the instrumentation.. move up to the ctor?

Reply:
👍